libs/ajax/jquery-ajax-unobtrusive/package.json

```diff
@@ -0,0 +1,50 @@
+{
+  "name": "jquery-ajax-unobtrusive",
+  "description": "Add-on to jQuery Ajax to enable unobtrusive options in data-* attributes",
+  "main": "dist/jquery.unobtrusive-ajax.js",
```

Comment:
```suggestion
```

We don't use the `main` attr on cdnjs -- we use the `filename` attr which is included later in this file.